src/roman_datamodels/datamodels/_datamodels.py

```diff
@@ -156,6 +156,13 @@ def __init__(self, init=None, **kwargs):
 
         if init is not None:
             self.meta.model_type = self.__class__.__name__
+            self.meta.calibration_software_name = "RomanCAL"
+
+            if "file_date" not in self.meta:
+                self.meta.file_date = Time.now()
+
+            if "origin" not in self.meta:
+                self.meta.origin = "STSCI/SOC"
```

Comment:
```suggestion
```
Let's handle this in the mixin classes. The code here populates/overwrites these values for all models. With this branch if I save a model where I've set `calibration_software_name` to a different value it incorrectly reads back in as `RomanCal`.

Reply:
This will require a mixin for every datamodel that isn't a reference file.

Reply:
You understand that code better than I do. So feel free to push a solution to my branch.

Reply:
I think I have a solution by passing "default set of defaults" for the datamodel products.